.env.test

```diff
@@ -6,4 +6,10 @@ AWS_REGION=us-east-1
 S3_DIRECT_UPLOAD_BUCKET=et3directbuckettest
 AWS_ENDPOINT=http://localhost:${S3_PORT}
 AWS_S3_FORCE_PATH_STYLE=true
-SELENIUM_PORT=4444
\ No newline at end of file
+SELENIUM_PORT=4444
+AZURE_STORAGE_BLOB_PORT=10000
```

Comment:
As you are tied in to using this code with the emulator, there is no need to use environment variables - you can hard code in the values ?

Reply:
I already had done but left the env vars there just in case - I have removed them in 7a661a0.